packages/lib/src/vis/shared/PanZoomMesh.tsx

```diff
@@ -90,8 +103,11 @@ function PanZoomMesh() {
   const onWheel = useCallback(
     (evt: ThreeEvent<WheelEvent>) => {
       const { sourceEvent, unprojectedPoint } = evt;
-      const factor = sourceEvent.deltaY > 0 ? ZOOM_FACTOR : 1 / ZOOM_FACTOR;
+      if (!isEventValid(sourceEvent, modifierKey)) {
+        return;
+      }
```

Comment:
I added the check for `modifierKey` for the wheel event as well. Not sure if that is worth it ? :shrug: